### PR TITLE
Opera iOS post

### PR DESCRIPTION
**Pull request by autonome**

### CODE REVIEW

File: content/post/opera-ios.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<p style="text-align: center;">
+  <img src="/img/opera-ipfs-header.png" alt="IPFS in Opera Touch for iOS">
+</p>
+
+
+In 2020 we announced a big moment for IPFS: The first official support of IPFS protocol addressing in a major browser, when [Opera released IPFS support in their Android browser](ipns://blog.ipfs.io/2020-03-30-ipfs-in-opera-for-android/ "Opera Android IPFS announcement"). This was an important step in IPFS browser support generally, by building interest and momentum. We didn’t stop there, and we’ve got not just one… BUT TWO releases to share with you today:
```
Comment (lidel) [inline content/post/opera-ios.md:6]: I'd love to use `ipns://` by default, but we are not there yet :-D 
```suggestion
In 2020 we announced a big moment for IPFS: The first official support of IPFS protocol addressing in a major browser, when [Opera released IPFS support in their Android browser](https://blog.ipfs.io/2020-03-30-ipfs-in-opera-for-android/). This was an important step in IPFS browser support generally, by building interest and momentum. We didn’t stop there, and we’ve got not just one… BUT TWO releases to share with you today:
```

File: content/post/opera-ios.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+First, Opera has now added support for IPFS addressing to Opera Touch, their iOS browser.
+
+<p style="text-align: center;">
+  <img src="/img/opera-ios-wikipedia-short.png" alt="A screenshot of Wikipedia on IPFS in Opera Touch">
+</p>
+
+Second, support for IPFS addressing in Opera desktop browser for Windows, macOS and Linux will be coming in their next release, currently planned for March 2021.
+
+With these releases, Opera will now support ipfs:// and ipns:// addressing across their browser product line on all major operating systems: Windows, macOS, Linux, iOS and Android.
```
Comment (lidel) [inline content/post/opera-ios.md:16]:
```suggestion
With these releases, Opera will now support `ipfs://` and `ipns://` addressing across their browser product line on all major operating systems: Windows, macOS, Linux, iOS and Android.
```

File: content/post/opera-ios.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+## How Do I use it?
+
+First, install Opera Touch on your iOS device. If you’re reading this on an iOS device, [click to install now](https://apps.apple.com/us/app/opera-touch-web-browser/id1411869974)
+
+<p style="text-align: center;">
+  <img src="/img/opera-ios-app-store-short.png" alt="Opera Touch in the iOS app store">
+</p>
+
+Then you can open content using IPFS protocol addresses, like the [ipns://blog.ipfs.io](blog you're reading now), [Wikipedia](ipns://en.wikipedia-on-ipfs.org/wiki/), or this [Persian room guardian](ipfs://bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi/). You can easily upload files to IPFS using the [IPFS desktop application](https://docs.ipfs.io/install/ipfs-desktop/) or [Share.ipfs.io](https://share.ipfs.io].
```
Comment (lidel) [inline content/post/opera-ios.md:26]: I am afraid Hugo will ignore non-http URIs in markdown, but `<a>` tags should work?
Also, maybe shoutout to Anya?

```suggestion
Then you can open content using IPFS protocol addresses, like the <a href="ipns://blog.ipfs.io">blog you're reading now</a>, <a href="ipns://en.wikipedia-on-ipfs.org">Wikipedia</a>, or this <a href="ipfs://bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi">Persian room guardian</a> by [Anya Boz](https://www.anyabozartist.com/the-persian-cat). You can easily upload files to IPFS using the [IPFS desktop application](https://docs.ipfs.io/install/ipfs-desktop/) or [Share.ipfs.io](https://share.ipfs.io).
```

File: content/post/opera-ios.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+## How Does it Work?
+
+Opera Touch supports navigating to addresses for ipfs:// and ipns:// protocol schemes, which are handled by a remote HTTP gateway. By default the gateway used is dweb.link, which is operated by Protocol Labs. TODO: Can select/change gateway?
```
Comment (lidel) [inline content/post/opera-ios.md:30]:
```suggestion
Opera Touch supports navigating to addresses for `ipfs://` and `ipns://` protocol schemes, which are handled by a remote HTTP gateway. By default the gateway used is `dweb.link`, which is operated by Protocol Labs. TODO: Can select/change gateway?
```

File: content/post/opera-ios.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+Native representation of IPFS addresses in browsers even when the content is loaded from an HTTP gateway as 1) it communicates to users the integrity guarantees built into the IPFS protocol and 2) it lets developers use IPFS addresses even as local node support is not yet implemented in most browsers.
```
Comment (lidel) [inline content/post/opera-ios.md:32]:
```suggestion
Native representation of IPFS addresses in browsers even when the content is loaded from an HTTP gateway as 1) it communicates to users the integrity guarantees built into the IPFS protocol and 2) it lets developers use IPFS addresses even as local node support is not yet implemented in most browsers.
```

I am not sure if I agree with this entire paragraph:
- (1) is plainly untrue (there are no integrity guarantees atm, user delegates trust to third party gateway). 
- (2) is is vague on added value. As Brave demonstrated you can resolve native URIs without keeping them in address bar. 

Can we rewrite it and  focus on less vague statements, like the fact that IPFS URIs are resolved out of the box, without the need for any additional extensions or opt-in settings, which familiarizes users and developers with decentralized concepts of content-addressing?

**Review by lidel — CHANGES_REQUESTED** [review]
Thank you @autonome for kicking this off! 

- [ ] Needs proper header
- [ ] Needs images
- [ ] Check output produced by Hugo

+ added some comments/suggestions inline.

File: content/post/opera-ios.md (hunk 1 above)
Comment (autonome) [reply content/post/opera-ios.md:6]: OMG this is what happens when u have IPFS support by default in a browser ;)

File: content/post/opera-ios.md (hunk 5 above)
Comment (autonome) [reply content/post/opera-ios.md:32]: Hm, we should figure out language for what integrity guarantees mean. I think we're talking about different things here. I'll remove entirely and replace with your suggestion for now!

**Comment by autonome** [comment]
@lidel All the images and header are already landed. Those paths resolve once published.

**Comment by autonome** [comment]
@lidel Fixed the "why native protocol addressing + remote http gateway is important" paragraph, can you review?

File: content/post/opera-ios.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
+<p style="text-align: center;">
```
Comment (jennwrites) [inline content/post/opera-ios.md:1]: TODO: Front matter plz

File: content/post/opera-ios.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+<p style="text-align: center;">
+  <img src="/img/opera-ios-app-store-short.png" alt="Opera Touch in the iOS app store">
+</p>
+
+Then you can open content using IPFS protocol addresses, like the <a href="ipns://blog.ipfs.io">blog you’re reading now</a>, <a href="ipns://en.wikipedia-on-ipfs.org">Wikipedia</a>, or this <a href="ipfs://bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi">Persian room guardian</a> by [Anya Boz](https://www.anyabozartist.com/the-persian-cat). You can easily upload files to IPFS using the [IPFS desktop application](https://docs.ipfs.io/install/ipfs-desktop/) or [Share.ipfs.io](https://share.ipfs.io).
+
+## How does it work?
+
+Opera Touch supports navigating to addresses for `ipfs://` and `ipns://` protocol schemes, which are handled by a remote HTTP gateway. By default the gateway used is `dweb.link`, which is operated by Protocol Labs. TODO: Can select/change gateway?
```
Comment (jennwrites) [inline content/post/opera-ios.md:30]: TODO: Resolve this TODO line ☺️

Comment (mandrigin) [reply content/post/opera-ios.md:30]: Not in 2.5.5, should be released with 2.5.6 hopefully this week (release testing ongoing).

File: content/post/opera-ios.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+Native representation of IPFS addresses in browsers is important, even when the content is loaded from an HTTP gateway as IPFS URIs are resolved out of the box, without the need for any additional extensions or opt-in settings, which familiarizes users and developers with decentralized concepts of content-addressing and creates a path to readiness for when native nodes are available.
+
+## What’s next?
+
+Opera desktop browser support for IPFS addressing will ship soon, and we’re discussing what additional features to add next to build on top of this foundation of universal addressing support across the Opera browser line. Have ideas for what you’d like to see in Opera next? Let us know on the [IPFS forums](https://discuss.ipfs.io/), or tweet your ideas to [@Opera](https://twitter.com/opera) and [@IPFS](https://twitter.com/ipfs) on Twitter!
```
Comment (mandrigin) [inline content/post/opera-ios.md:36]: Anything about the preview builds on `ethdenver2021.opera.com`?

Comment (autonome) [reply content/post/opera-ios.md:36]: Sure! Doesn't load yet?

File: content/post/opera-ios.md (hunk 6 above)
Comment (autonome) [reply content/post/opera-ios.md:13]: added!

File: content/post/opera-ios.md (hunk 7 above)
Comment (autonome) [reply content/post/opera-ios.md]: Fixed!

File: content/post/opera-ios.md (hunk 8 above)
Comment (autonome) [reply content/post/opera-ios.md:50]: Added!

**Pull request merged**